app/views/layouts/_sidebar.html.erb

```diff
@@ -83,7 +83,7 @@
         </li>
 
         <li class="sidebar-item  ">
-          <a href="application-chat.html" class='sidebar-link'>
+          <a href="<%= users_path%>" class='sidebar-link'>
```

Comment:
We should probably use a `link_to` rather than an href... something like:
```ruby
<%= link_to users_path, class: 'sidebar-link' do %>
  <i class="bi bi-people"></i>
  <span>Users</span>
<% end %>
```